config/locales/en.yml

```diff
@@ -30,4 +30,4 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  service_name: "Teachers: claim back student loan repayments"
```

Comment:
Just asked the team here what service name we should start with, it was suggested we go with `Teacher’s Payment Service`